.github/CONTRIBUTING.md

```diff
@@ -71,6 +70,18 @@ Explain local setup
 Explain CI setup
 ```
 
+In most cases the `<type>` should be one of the following:
```

Comment:
I'm personally not a fan of trying to limit commits per type.

Let's take an example - I fix a bug with a new extracted method. That new method naturally should be documented, and ideally, tests written for it.

IMO, that should all be under a single commit - it's an atomic piece of work - so it could potentially be rolled back. Just rolling back the method without the docs or tests wouldn't make sense. 

As such, trying to add a `<type>` to that single commit is going to be wrong, since it (rightly) covers multiple types. If it can't be right, and it can't be applied consistently, there's no benefit to having it since the git log can't be trusted.